sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -147,13 +150,14 @@ int ng_ipv6_netif_add_addr(kernel_pid_t pid, const ng_ipv6_addr_t *addr,
         return -ENOENT;
     }
 
-    if ((addr == NULL) || (ng_ipv6_addr_is_unspecified(addr))) {
+    if ((addr == NULL) || (ng_ipv6_addr_is_unspecified(addr)) ||
+        (prefix_len < 1) || (prefix_len > 128)) {
```

Comment:
`prefix_len < 1` is only checking the case for `prefix_len` being `0` (because it's defined as `uint8_t` and never becomes negative). You could replace this by `(!prefix_len) || (prefix_len > 128)`.
But I would prefer the (*maybe* more optimized) alternative: `((unsigned) x - 1 > 127)`. This will result in `true` for `x < 1` and `x > 128`.